sklearn/metrics/classification.py

```diff
@@ -313,8 +315,11 @@ class labels [2].
     .. [1] J. Cohen (1960). "A coefficient of agreement for nominal scales".
```

Comment:
[`precision_recall_fscore_support`](https://github.com/hlin117/scikit-learn/blob/8e0fcdf6898e2723987d6f45268c631b09e3e32b/sklearn/metrics/classification.py#L969) seems to use URL as well.

Overall, I guess having both DOI and URL is permissible. I don't have a preference for which one to use.